include/fast_float/float_common.h

```diff
@@ -30,7 +30,7 @@ inline bool fastfloat_strncasecmp(const char *input1, const char *input2,
 #error "FLT_EVAL_METHOD should be defined, please include cfloat."
 #endif
 
-bool is_space(uint8_t c) {
+inline bool is_space(uint8_t c) {
   static const bool table[] = {
```

Comment:
@lemire this is_space function is used only to remove leading zeroes at start of the parsing, but static table will be attached to every translation unit, maybe it make sense to implement it with simple 6 comparisons ?